code/src/types/product.types.ts

```diff
@@ -0,0 +1,9 @@
+export type Product = {
+  id: string;
+  name: string;
+  description: string;
+  status: "AVAILABLE";
```

Comment:
I suggest to make a separate type for status and use for product

Reply:
Fixed